src/App.css

```diff
@@ -1,14 +1,281 @@
+}
+
+.Circular {
+  display: flex;
+  position: relative;
+  right: 20%;
+  gap: 1.313rem;
+  flex-grow: 1;
+}
+
+.Circular .Spin {
+  width: 4.25rem;
+  height: 4.25rem;
+  border: solid 5px #379cf6;
+  border-width: 5px;
+  border-radius: 50%;
+  border-left-color: lightgray;
+  transform: rotate(45deg);
 }
 
-.Bookform {
-  margin-top: -10px;
-  margin-left: 35%;
+.Circular h2 {
+  width: 4.25rem;
+  height: 2.438rem;
+  position: absolute;
+  top: -15px;
+  font-family: Montserrat, sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.Circular p {
+  font-family: Montserrat, sans-serif;
+  opacity: 0.5;
+  position: relative;
+  top: 35px;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.Line3 {
+  width: 0.002rem;
+  height: 7.975rem;
+  position: absolute;
+  left: 130%;
+  border: solid 1px #e8e8e8;
+}
+
+.Current {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 0 0 0;
+  opacity: 0.5;
+  font-family: Roboto Slab, sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+}
+
+.ChapterNo {
+  width: 4.688rem;
+  height: 1.313rem;
+  font-family: Roboto Slab, sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: var(--black-two);
+}
+
+.Rectangle {
+  width: 11.5rem;
+  height: 2.063rem;
+  border-radius: 3px;
+  background-color: rgba(15, 127, 255, 0.637);
+}
+
+.Rectangle button {
+  font-family: Roboto Slab, sans-serif;
+  background: none;
+  color: #fff;
+  margin-left: 10px;
+}
+
+.AddBook {
+  padding: 1rem;
+  margin: 0 2.5rem 8rem 2.5rem;
+}
+
+.AddBookTitle {
+  margin: 4.913rem 32.75rem 1.188rem 6rem;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+input[type='text'] {
+  width: 10rem;
+  height: 2rem;
+  margin-left: 30px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+::placeholder {
+  font-size: 1rem;
+  padding-left: 1rem;
+  letter-spacing: -0.15px;
+  color: #c4c4c4;
+}
+
+.CategoryList {
+  width: 15rem;
+  height: 2rem;
+  border-radius: 5px;
+  border: solid 1px #e8e8e8;
+  padding: 0 0.5rem;
+  font-size: 1rem;
+  color: #c4c4c4;
+  letter-spacing: -0.15px;
+  margin-left: 2rem;
+}
+
+.AddBook button {
+  width: 14rem;
+  height: 2rem;
+  border-radius: 5px;
+  border: none;
+  padding: 0;
+  font-size: 0.8rem;
+  color: #fff;
+  background-color: #0290ff;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.BookForm {
```

Comment:
- [ ] Great job, kindly make sure the spacing for the form inputs resemble the design, also check the font size and font weight.
### Design
![image](https://user-images.githubusercontent.com/59416136/202177976-d1bf4d54-7b9f-433b-8e0c-f72db6d8579c.png)


### Your Implementation
<img width="984" alt="Screenshot 2022-11-16 at 3 06 43 PM" src="https://user-images.githubusercontent.com/59416136/202177570-9b09e8d2-e110-47ce-bc54-73cc9544dae9.png">

Reply:
Space is made accordingly.